src/main.tsx

```diff
@@ -12,6 +14,16 @@ const router = createBrowserRouter([
     path: "/",
     element: <App />,
     errorElement: <NotFound />
+  },
+  {
+    path: "/login",
+    element: <Login />,
+    errorElement: <NotFound />
```

Comment:
I'm not sure but I think that the `errorElement` property is only needed in the `"/"` path.